enable/savage/svg/svg_regex.py

```diff
@@ -114,9 +115,9 @@ def __init__(self, lexer=svg_lexer):
     def parse(self, text):
         """ Parse a string of SVG <path> data.
         """
-        next = self.lexer.lex(text).next
-        token = next()
-        return self.rule_svg_path(next, token)
+        svg_iterator = self.lexer.lex(text)
+        token = six.next(svg_iterator)
+        return self.rule_svg_path(svg_iterator, token)
```

Comment:
This appears to be incorrect. Perhaps something like: `return self.rule_svg_path(partial(six.next, svg_iterator), token)` or its equivalent. The first argument to `self.rule_svg_path` has to be a callable.